src/queue.jl

```diff
@@ -13,11 +22,28 @@ length(s::Queue) = length(s.store)
 front(s::Queue) = front(s.store)
 back(s::Queue) = back(s.store)
 
+"""
+    enqueue!(s::Queue, x)
+
+### Args:
+* `s`: A queue into which to insert a value. 
+* `x`: The value to be inserted.
+
+`enqueue!` inserts the value `x` into the queue `s`.
```

Comment:
Also, "insert" could be more explicit about whether it appends to the end of the queue, or to its start.